handler/src/main/java/io/netty/handler/pcap/TCPPacket.java

```diff
@@ -36,13 +36,13 @@ private TCPPacket() {
      * @param srcPort Source Port
      * @param dstPort Destination Port
      */
-    static void writePacket(ByteBuf byteBuf, ByteBuf payload, int segmentNumber, int ackNumber, int srcPort,
+    static void writePacket(ByteBuf byteBuf, ByteBuf payload, long segmentNumber, long ackNumber, int srcPort,
                             int dstPort, TCPFlag... tcpFlags) {
 
         byteBuf.writeShort(srcPort);     // Source Port
         byteBuf.writeShort(dstPort);     // Destination Port
-        byteBuf.writeInt(segmentNumber); // Segment Number
-        byteBuf.writeInt(ackNumber);     // Acknowledgment Number
+        byteBuf.writeInt((int) segmentNumber); // Segment Number
```

Comment:
I doubt this

Reply:
We should produce a 5G file and try to read it using Wireshark just to be double-sure.

Reply:
@BradleyJ-SAIC can you do this ?

Reply:
@normanmaurer I'll have time today to look into this and address any issues I find.

Reply:
Found 2 issues directly related to this PR while using Wireshark to look at the generated 5GB pcap. Those will be addressed.

Noticed two other issues present in 4.1.

1. If the payload is larger than 65495 for TCP, the length field in the IP packet is incorrect due to the length being a ushort. UDP will have a similar problem.
![image](https://github.com/user-attachments/assets/155cec08-767f-4efc-a2c5-c9c04718bd75)
2. When the PcapWriteHandler is attached to a TCP server, the FIN messages have incorrect segment and ack numbers. 
![image](https://github.com/user-attachments/assets/236c3109-558d-4fa2-8cce-5fa05503ebc0)


I will try to find the time to address these here with unit tests

Reply:
For TCP, the payload can just be split into multiple PCAP entries. 
For UDP, I imagine it could just log that it received too large of a payload.

Reply:
Alright, all of the above have been handled. Tested large PCAP files and those show correctly in Wireshark now.